### PR TITLE
Update UG, PPP, and Testing

### PR DESCRIPTION
**Pull request by longnguyentan**
Update UG, PPP, and Testing

## Key changes
- Update the Introduction and Find, Help, List, Clear, Exit features in UG.
- Update the contribution in PPP.
- Fix bugs in testing.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -6,43 +6,23 @@ pageNav: 3
...
-patient records to manage? Do you wish for a faster, more efficient way to log patient reports and streamline your
-workflow? A&E is the tool you’ve been waiting for!
+Welcome to **Advanced&Efficient (A&E)**!
 
-Advanced&Efficient (A&E) is a powerful **desktop application** designed specifically for **ED doctors** who need to *
-*create, update, and manage patient records** swiftly and effectively. This application, with its **Command-Line
-Interface (CLI)** design, caters to fast typists and professionals who prefer typing over mouse interactions,
-significantly speeding up the process of inputting and managing patient data.
+Advanced&Efficient (A&E) is a powerful **desktop application** designed specifically for **Emergency Department (ED) doctors** who need to **create, update, and manage patients and their records** swiftly and effectively. This application, with its [**Command-Line Interface (CLI)**](#cli) design, caters to fast typists and professionals who prefer typing over mouse interactions, significantly speeding up the process of inputting and managing patient data.
```
Comment (AaronJT1) [inline docs/UserGuide.md:11]: Nice job updating the intro!

File: docs/UserGuide.md
Comment (AaronJT1) [inline docs/UserGuide.md:1]: user guide looks good to me!

File: docs/team/longnguyentan.md [hunk 2]
```diff
@@ -19,9 +19,10 @@ Given below are my contributions to the project.
 * **Contributions to the UG**:
   * Wrote Introduction & resized images ([#165](https://github.com/AY2324S1-CS2103T-T14-2/tp/pull/165)
   * Updated `list` and `delete` features ([#51](https://github.com/AY2324S1-CS2103T-T14-2/tp/pull/51)
+  * Updated features: Find, Help, List, Clear, Exit
```
Comment (AaronJT1) [inline docs/team/longnguyentan.md:22]: Good that you included your contribution. Would it be helpful if you add links to pr that shows it? Not really sure how our TA will assess

File: src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java [hunk 3]
```diff
@@ -26,6 +26,9 @@ public void toModelType_typicalPatientsFile_success() throws Exception {
                 JsonSerializableAddressBook.class).get();
         AddressBook addressBookFromFile = dataFromFile.toModelType();
         AddressBook typicalPatientsAddressBook = TypicalPatients.getTypicalAddressBook();
+        System.out.println(addressBookFromFile);
+        System.out.println("-----------------------");
+        System.out.println(typicalPatientsAddressBook);
```
Comment (AaronJT1) [inline src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java:31]: Are these comments for debugging purposes or intended? If its intended then its fine!!

Comment (thienmy0) [inline src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java:31]: Hi! May I ask what this is for?

**Review by AaronJT1 — APPROVED** [review]

File: docs/UserGuide.md
Comment (thienmy0) [inline docs/UserGuide.md:1]: Thanks for updating the DG!

**Review by thienmy0 — COMMENTED** [review]
Looks good overall!

File: src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java (hunk 3 above)
Comment (longnguyentan) [reply src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java:31]: It's for debugging as I modified the JSON file

Comment (longnguyentan) [reply src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java:31]: It's for debugging as I modified the JSON file

File: docs/team/longnguyentan.md (hunk 2 above)
Comment (longnguyentan) [reply docs/team/longnguyentan.md:22]: I'll update it when it's close to the deadline

**Pull request merged**